frontend/app-development/features/SettingsModalButton/SettingsModal/SettingsModal.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { ReactNode, useState } from 'react';
+import classes from './SettingsModal.module.css';
+import { Heading } from '@digdir/design-system-react';
+import { CogIcon, InformationSquareIcon, ShieldLockIcon } from '@navikt/aksel-icons';
+import { Modal } from 'app-shared/components/Modal';
+import { LeftNavigationTab } from 'app-shared/types/LeftNavigationTab';
+import { LeftNavigationBar } from 'app-shared/components/LeftNavigationBar';
+import { SettingsModalTab } from 'app-development/types/SettingsModalTab';
+import { createNavigationTab } from './utils';
+import { useTranslation } from 'react-i18next';
+import { Policy } from '@altinn/policy-editor';
+import { PolicyTab } from './components/Tabs/PolicyTab';
+
+export type SettingsModalProps = {
+  /**
+   * Flag for if the modal is open
+   */
+  isOpen: boolean;
+  /**
+   * Function to be executed on close
+   * @returns void
+   */
+  onClose: () => void;
+  /**
+   * The policy of the app
+   */
+  policy: Policy;
+  /**
+   * The org
+   */
+  org: string;
+  /**
+   * The app
+   */
+  app: string;
+};
+
+/**
+ * @component
+ *    Displays the settings modal.
+ *
+ * @property {boolean}[isOpen] - Flag for if the modal is open
+ * @property {function}[onClose] - Function to be executed on close
+ * @property {Policy}[policy] - The policy of the app
+ * @property {string}[org] - The org
+ * @property {string}[app] - The app
+ *
+ * @returns {ReactNode} - The rendered component
+ */
+export const SettingsModal = ({
+  isOpen,
+  onClose,
+  policy,
+  org,
+  app,
+}: SettingsModalProps): ReactNode => {
+  const { t } = useTranslation();
+
+  const [currentTab, setCurrentTab] = useState<SettingsModalTab>('about');
+
+  /**
+   * Ids for the navigation tabs
+   */
+  const aboutTabId: SettingsModalTab = 'about';
+  const policyTabId: SettingsModalTab = 'policy';
+
+  /**
+   * The tabs to display in the navigation bar
+   */
+  const leftNavigationTabs: LeftNavigationTab[] = [
+    createNavigationTab(
+      <InformationSquareIcon className={classes.icon} />,
+      aboutTabId,
+      () => changeTabTo(aboutTabId),
+      currentTab
+    ),
+    createNavigationTab(
+      <ShieldLockIcon className={classes.icon} />,
+      policyTabId,
+      () => changeTabTo(policyTabId),
+      currentTab
+    ),
+  ];
+
+  /**
+   * Changes the active tab
+   * @param tabId
+   */
+  const changeTabTo = (tabId: SettingsModalTab) => {
+    setCurrentTab(tabId);
+  };
+
+  const displayTabs = () => {
+    switch (currentTab) {
+      case 'about': {
+        return <p> {currentTab}</p>;
```

Comment:
Er dette noe som skal implementeres senere?

Reply:
Ja. Grunnen til at den er lagt inn nå er for å kjøre logikken og se at det funker. Den var inne i master før denne PRen, men ble flyttet

Reply:
Vi bør ha en mer forklarende tekst i så fall. "Denne siden er under utvikling og vil bli publisert på et senere tidspunkt" eller noe slikt.

Reply:
Ja sånn ja. Enig 😄